Postman/Nfield Public API AAD.postman_collection.json

```diff
@@ -16547,6 +16547,345 @@
+									"value": "{{BearerToken}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "X-Nfield-Domain",
+								"value": "{{DomainTestName}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{origin-public-api}}/v1/Events/Subscriptions",
+							"host": [
+								"{{origin-public-api}}"
+							],
+							"path": [
+								"v1",
+								"Events",
+								"Subscriptions"
+							]
+						},
+						"description": "Gets a list of repository plans."
+					},
+					"response": []
+				},
+				{
+					"name": "Get Subscription",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"protocolProfileBehavior": {
+						"disableBodyPruning": true
+					},
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{BearerToken}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "X-Nfield-Domain",
+								"value": "{{DomainTestName}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{origin-public-api}}/v1/Events/Subscriptions/:SubscriptionName",
+							"host": [
+								"{{origin-public-api}}"
+							],
+							"path": [
+								"v1",
+								"Events",
+								"Subscriptions",
+								":SubscriptionName"
+							],
+							"variable": [
+								{
+									"key": "SubscriptionName",
+									"value": "{{EventSubscriptionName}}"
+								}
+							]
+						},
+						"description": "Gets a list of repository plans."
+					},
+					"response": []
+				},
+				{
+					"name": "Create Subscription",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status code is 201\", function () {\r",
+									"    pm.response.to.have.status(201);\r",
+									"});\r",
+									"\r",
+									"const validateProperties = (eventSubscription) => {\r",
+									"    return eventSubscription.EventSubscriptionName !== undefined\r",
+									"    && eventSubscription.Endpoint !== undefined\r",
+									"    && eventSubscription.EventTypes !== undefined;\r",
+									"}\r",
+									"\r",
+									"pm.test(\"Validate properties and set Event Subscription Name\", function () {\r",
+									"    const office = pm.response.json();\r",
```

Comment:
Why this line is needed?

Reply:
This line shouldn't be there as it has been deleted! I will re-upload the exported docs.